searchlib/src/vespa/searchlib/attribute/attribute_blueprint_factory.cpp

```diff
@@ -139,6 +140,18 @@ class AttributeFieldBlueprint : public SimpleLeafBlueprint
                             QueryTermSimple::UP term, const SearchContextParams &params);
     ~AttributeFieldBlueprint() override;
 
+    search::queryeval::FlowStats calculate_flow_stats(uint32_t docid_limit) const override {
+        double rel_est = abs_to_rel_est(getState().estimate().estHits, docid_limit);
```

Comment:
no need to calculate this if we do not know, should be moved to else case. Also, no need to ask the state for the value back since we already have the value stored directly as a member.

Reply:
Fixed. PTAL